bin/ci2gubernator.sh

```diff
@@ -35,9 +35,9 @@ go get -u istio.io/test-infra/toolbox/ci2gubernator
 
 /go/bin/ci2gubernator ${@} \
 --service_account="${TMP_SA_JSON}" \
---sha=${CIRCLE_SHA1} \
+--sha=${CIRCLE_BRANCH}/${CIRCLE_SHA1} \
```

Comment:
Could you explain why the branch prefix is required here? I am concerned that it might have consequences to downstream processing of test results on GCS if they expect actual vanilla SHA.